UIAlertController+Show/UIAlertController+Show.swift

```diff
@@ -24,15 +24,19 @@ extension UIAlertController {
     }
 
     @objc public func show() {
-        show(true)
+        show(true, completion: nil)
```

Comment:
because nil is the default parameter valuer for the completion closure, can this remain unchanged as `show(true)`?